CMakeLists.txt

```
@@ -553,8 +553,13 @@ if(NRN_ENABLE_TESTS)
   include(CTest)
   include(Catch)
   find_python_module(pytest)
-  if(NRN_ENABLE_PYTHON AND NOT PYTEST_FOUND)
-    message(SEND_ERROR "pytest Python package is required.")
+  find_python_module(pytest-cov)
+  if(NRN_ENABLE_PYTHON)
+    if(NOT PYTEST_FOUND)
+      message(SEND_ERROR "pytest Python package is required.")
+    elseif(NOT PYTEST_COV_FOUND)
+      message(WARNING "pytest-cov not installed. No coverage will be provided.")
```

Small nitpick is that something like:
```suggestion
      message(WARNING "pytest-cov not installed. Test coverage reports will not be generated.")
```
would be more descriptive IMO.